_data/tool_and_resource_list.yml

```diff
@@ -1,3 +1,55 @@
+- description: scikit-learn is a free software machine learning library for the Python programming language.
+  id: scikit-learn
+  name: scikit-learn
+  url: https://scikit-learn.org
+  registry:
+    biotools: scikit-learn
+- description: The CUDA Toolkit from NVIDIA provides everything you need to develop GPU-accelerated applications
+  id: cuda
+  name: CUDA toolkit
+  url: https://developer.nvidia.com/cuda-toolkit
+- description: A free and open-source software library for machine learning and artificial intelligence.
+  id: tensorflow
+  name: Tensorflow
+  url: https://www.tensorflow.org
+  registry: 
+    biotools: tensorflow
+- description: Keras is an open-source library that provides a Python interface for artificial neural networks.
+  id: keras
+  name: Keras
+  url: https://keras.io/
+- description: BioImage is a collaborative effort to bring AI models to the bioimaging community. 
+  id: bioimage
+  name: BioImage Model Zoo
+  url: https://bioimage.io/#/
+- description: Schema.org is a collaborative, community activity with a mission to create, maintain, and promote schemas for structured data on the Internet, on web pages, in email messages, and beyond.
```

Comment:
schema.org is already added in the tool list as "schema-org"

Reply:
@bedroesb  Ok, I will fix this.